Formula/clamav.rb

```diff
@@ -44,6 +45,7 @@ class Clamav < Formula
   def install
     args = std_cmake_args + %W[
       -DAPP_CONFIG_DIRECTORY=#{etc}/clamav
+      -DDATABASE_DIRECTORY=#{var}/clamav
```

Comment:
Maybe `var/"cache"` or `var/"run"`?

Should we run `freshclam` in `postinstall` if this directory is empty?

Reply:
`var/lib/clamav` is the most common (e.g. Debian and Arch use this). It is also part of example config:
```
#DatabaseDirectory /var/lib/clamav
#UpdateLogFile /var/log/freshclam.log
#PidFile /var/run/freshclam.pid
---
#PidFile /var/run/clamd.pid
#TemporaryDirectory /var/tmp
#DatabaseDirectory /var/lib/clamav
```

---

Homebrew does use `var/#{name}` for (my|postgre)SQL DBs but `var/lib/#{name}` for InfluxDB and Opensearch

Reply:
> Maybe `var/"cache"` or `var/"run"`?

Hmmmm, probably the former. The latter is for ephemeral data like PID files, right?

> Should we run `freshclam` in `postinstall` if this directory is empty?

That wouldn't work, because the user has to create a `freshclam.conf` appropriate to their intended usage anyway. Without that config file, `freshclam` will abend, and not configuring and running `freshclam` is a user error IMO (no AV updates otherwise).

Reply:
Presumably the location of the `pid` file is also configurable, maybe we also want to set that to `var/"run"`.

Reply:
Default behavior is no PID file. Looks to be modifiable via runtime .conf file rather than a build-time configuration option.